siliconcompiler/targets/asic_demo.py

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
This needs be added to the quick CI

Reply:
We're coming down to the wire on release timing, but sure, shouldn't take too long.